Update readme about .NET 3.5 scripting runtime on UWP (#427)
Microsoft/AppCenter-SDK-Unity: CHANGELOG.md

### App Center Distribute

* **[Feature]** Add a **Automatic Check For Update** checkbox that can be unchecked to turn off automatic check for update.
* **[Feature]** Add a `CheckForUpdate` API to manually check for update.

> ## Release 3.0.0

Reviewer: 3.0.0 is not release, it will be one PR

### App Center

* **[Breaking change]** Bump the minimal supported Unity version from **5.6** to **2018.1**.

> * **[Breaking change]** Drop .NET 3.5 scripting runtime version.

Reviewer: This is not true. It doesn't supported on UWP only

* **[Feature]** Add `UpdateTrack` property to be able to explicitly set either `Private` or `Public` update track. By default, a public distribution group is used. **Breaking change**: To allow users to access releases of private groups you now need to migrate your application to call `Distribute.UpdateTrack = UpdateTrack.Private` before the SDK start. Please read the documentation for more details.
* **[Behavior change]** The public distribution is simplified to provide only one public group. If you have existing public groups defined for your application your users will receive the latest version of all public groups.

> * **[Feature]** Add a **Automatic Check For Update** checkbox that can be unchecked to turn off automatic check for update.

Reviewer 2: ```suggestion
* **[Feature]** Add an **Automatic Check For Update** checkbox that can be used to turn off automatic check for update.
```

Reviewer: It's on develop and not only in Unity repo, the PR just move it to right section
